src/registrar/tests/common.py

```diff
@@ -462,7 +462,7 @@ def completed_application(
 ):
     """A completed domain application."""
     if not user:
-        user = get_user_model().objects.create(username="username")
+        user = get_user_model().objects.create(username="username" + str(uuid.uuid4())[:8])
```

Comment:
Thats interesting that this one had to be modified, too. This class was introduced before the big one in common - this is only an issue now. I have the hunch that there is some insidious bug sneaking around causing this (probably related to or the same as that one we both discovered)